lib/usb/usb_f103.c

```diff
@@ -41,6 +42,7 @@ static uint16_t stm32f103_ep_write_packet(usbd_device *usbd_dev, uint8_t addr,
 static uint16_t stm32f103_ep_read_packet(usbd_device *usbd_dev, uint8_t addr,
 					 void *buf, uint16_t len);
 static void stm32f103_poll(usbd_device *usbd_dev);
+static void stm32f103_enable_sof_callback(usbd_device *dev);
```

Comment:
I do feel, personally, that as a backend method name, that this should either be, "enable sof interrupt" or "enable interrupt(sof)"  but it has nothing to do with a callback in this layer right?